dask_match/expr.py

```diff
@@ -117,6 +117,8 @@ def __getattr__(self, key):
             if key in _parameters:
                 idx = _parameters.index(key)
                 return self.operands[idx]
+            if isinstance(self._meta, pd.DataFrame) and key in self._meta.columns:
```

Comment:
I'd like to avoid pandas-specific checks, as one of my next highest priorities is to make sure this all works smoothly with a cudf backend.

Reply:
i.e. can we use `is_dataframe_like` here?

Reply:
Done.